modules/govuk_jenkins/templates/jobs/copy_data_to_integration.yaml.erb

```diff
@@ -50,8 +50,19 @@
             sed -i.bak "s/PG_TR_SRC_ENV_SYNC_PW=PLACEHOLDER/PG_TR_SRC_ENV_SYNC_PW='<%= @pg_src_env_sync_pw %>'/" scripts/sync-transition-postgresql.sh
             sed -i "s/PG_TR_DST_ENV_SYNC_PW=PLACEHOLDER/PG_TR_DST_ENV_SYNC_PW='<%= @pg_tr_dst_env_sync_pw %>'/" scripts/sync-transition-postgresql.sh
 
+            echo "Disabling Gor traffic replay"
+            for box in $(govuk_node_list -C 'govuk_gor'); do
```

Comment:
Assuming this relates to the job running from the Jenkins machine in the Carrenza side of Production, I don't think this code will do anything, as `govuk_node_list -C 'govuk_gor'` doesn't output anything.

Reply:
Aah thank you @cbaines , I was looking in AWS not Carrenza 🤦‍♀